src/main/kotlin/io/github/mojira/arisa/modules/ReopenAwaitingModule.kt

```diff
@@ -30,9 +31,7 @@ class ReopenAwaitingModule(
                 .created
             val lastComment = comments.last()
             assertGreaterThan(lastComment.created.toEpochMilli(), resolveTime.toEpochMilli()).bind()
-            assertUpdateWasNotCausedByEditingComment(
-                updated.toEpochMilli(), lastComment.updated.toEpochMilli(), lastComment.created.toEpochMilli()
-            ).bind()
+            assertGreaterThan(lastComment.created.toEpochMilli(), lastRun.toEpochMilli()).bind()
```

Comment:
[Minor] Not sure why the line above doesn't use it either, but there is also `assertIsAfter` which works for Instance